### ISSUE TITLE
1.1.4 breaks angular 8

### ISSUE THREAD

**Issue by brendanfmartin** [dmacfarlane/angular-mentions#144]
We updated to version `1.1.4` and are using Angular 8. With this breaking change can we publish the package under a version `2.0.0` or `1.2.0` and remove `1.1.4`?

ExpressionChangedAfterItHasBeenCheckedError

```
   "@angular/animations": "~8.2.14",
    "@angular/cdk": "~8.2.3",
    "@angular/common": "^8.2.14",
    "@angular/compiler": "^8.2.14",
    "@angular/core": "^8.2.14",
    "@angular/forms": "^8.2.14",
    "@angular/platform-browser": "^8.2.14",
    "@angular/platform-browser-dynamic": "^8.2.14",
    "@angular/router": "^8.2.14",
    "@datadog/browser-rum": "latest",
    "angular-animate": "^1.7.2",
    "angular-mentions": "^1.1.4",

```

```
<input type="text" [mention]="items">
```

```
items: string[] = ["Noah", "Liam", "Mason", "Jacob", ...
```

**Comment by dmacfarlane** [comment]
It was not intended to be a breaking change. I've published 1.1.5 which fixes this issue, but undoes the fix for the angular 9 issue in #141.

**Comment by brendanfmartin** [comment]
Thank you @dmacfarlane!
